### PR TITLE
Allow default block for `contentOf`

### PR DESCRIPTION
**Pull request by ruudk**
Fixes https://github.com/gobuffalo/plush/issues/57

### CODE REVIEW

**Comment by matryer** [comment]
Nice work @ruudk - sorry for breaking it.

File: content_helper.go [hunk 1]
```diff
@@ -38,7 +38,16 @@ func contentForHelper(name string, help HelperContext) {
 func contentOfHelper(name string, data map[string]interface{}, help HelperContext) (template.HTML, error) {
 	fn, ok := help.Value("contentFor:" + name).(func(data map[string]interface{}) (template.HTML, error))
 	if !ok {
-		return template.HTML(""), errors.WithStack(errors.New("missing contentOf block: " + name))
+		ctx := help.New()
+		for k, v := range data {
+			ctx.Set(k, v)
+		}
+		body, err := help.BlockWith(ctx)
```
Comment (markbates) [inline content_helper.go]: The block shouldn't be required. Currently this code will return an error if there is no block, which isn't the expected behavior of `contentOf`. You should check to see if there is a block first, before trying to run it.

Comment (ruudk) [reply content_helper.go]: Hmm, but how come the tests pass?

Comment (ruudk) [reply content_helper.go]: Fixed

**Comment by ruudk** [comment]
What's needed to get this merged?

**Pull request merged**